src/hotspot/share/ci/ciMethod.cpp

```diff
@@ -1278,6 +1286,18 @@ BCEscapeAnalyzer  *ciMethod::get_bcea() {
 #endif // COMPILER2
 }
 
+BCReachabilityAnalyzer  *ciMethod::get_bcra() {
+#ifdef COMPILER2 //Static Analyzer
```

Comment:
I guess we don't need to protect this code with `COMPILER2`.

Reply:
Agreed.  I've commented out the #else branch as we'll want effectively the same code using our own static analyzer define in the future